src/main/java/cubicchunks/asm/mixin/fixes/common/MixinBlockFalling_HeightLimits.java

```diff
@@ -0,0 +1,59 @@
+package cubicchunks.asm.mixin.fixes.common;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+import cubicchunks.world.ICubicWorld;
+import mcp.MethodsReturnNonnullByDefault;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockFalling;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.item.EntityFallingBlock;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+@MethodsReturnNonnullByDefault
+@ParametersAreNonnullByDefault
+@Mixin(BlockFalling.class)
+public abstract class MixinBlockFalling_HeightLimits extends Block {
+
+    public MixinBlockFalling_HeightLimits(Material materialIn) {
+        super(materialIn);
+    }
+
+    @SuppressWarnings("unused")
+    private void checkFallable(World worldIn, BlockPos pos) {
```

Comment:
First, for that thing you would use `@Overwrite` so that it works after reobfuscation.

Second, I'm almost sure you can do that with some redirects and injections, which are preferred over complete overwrite (when vanilla code changes in incompatible way, it's easier to find the error).